{{cookiecutter.project_repo}}/liccheck.ini

```diff
@@ -0,0 +1,43 @@
+# Authorized and unauthorized licenses in LOWER CASE
+[Licenses]
+authorized_licenses:
+        3-clause bsd
+        apache
+        apache 2.0
+        apache license 2.0
+        apache software
+        apache software license
+        apache-software
+        asl
+        asl 2
+        bsd
+        bsd license
+        bsd-3-clause
+        cc-by
+        gnu lgpl
```

Comment:
lesser is certainly better, but I would always suggest some level of documentation around all the licenses being used, that that often helps people to understand where the issues are.

Are there only 2 levels? could you have an review_authorised?

Reply:
The way to force it to be reviewed is to remove it from the approved list and add individual packages as part of the review process in the `[Authorized Packages]` sections

Reply:
I think so, does that feel ok. I'm not too precious of my views on which licences we could allow, but I'm always cautious around the GNU ones.